core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-list/dot-categories-list.component.ts

```diff
@@ -16,7 +16,9 @@ import { DotCategoriesListState, DotCategoriesListStore } from './store/dot-cate
     providers: [DotCategoriesListStore]
 })
 export class DotCategoriesListComponent {
-    vm$: Observable<DotCategoriesListState> = this.store.vm$;
+    readonly #store = inject(DotCategoriesListStore);
```

Comment:
Why modify this if is an upgrade of storybook?

Reply:
Was mistaken